config/routes.rb

```diff
@@ -71,12 +82,9 @@
       resource :content_block, only: %i[create]
     end
 
-    resources :courses, only: %i[index show new] do
+    resources :courses, only: %i[] do
```

Comment:
Now that there are no _resourceful_ actions here, this might as well be a namespace:

```suggestion
    namespace :courses do
```